src/compiler/utilities.ts

```diff
@@ -2615,11 +2620,19 @@ namespace ts {
         });
     }
 
-    /** Get the type annotaion for the value parameter. */
-    export function getSetAccessorTypeAnnotationNode(accessor: SetAccessorDeclaration): TypeNode {
+    /** Get the type annotation for the value parameter. */
+    export function getSetAccessorTypeAnnotationNode(accessor: SetAccessorDeclaration, includeJSDocType?: boolean): TypeNode {

```

Comment:
I considered actually making this a separate function, as `getSetAccessorTypeAnnotationNode` very clearly states its getting the type annotation, not the jsdoc comment. I don't want to just blindly grab the jsdoc comment.